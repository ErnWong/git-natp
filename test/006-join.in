A....
     >
B'''' \
C------+--F---G
     .'      .'.
D---'  .'''''   ''''-.
      E         H---->J
                     /
                    I
