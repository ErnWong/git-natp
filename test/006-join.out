A
B
C
D
E
F C A B D
G F E
H
I
J H I
